hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/FileSystemConfigurations.java

```diff
@@ -41,6 +41,7 @@ public final class FileSystemConfigurations {
   public static final int DEFAULT_MIN_BACKOFF_INTERVAL = 3 * 1000;  // 3s
   public static final int DEFAULT_MAX_BACKOFF_INTERVAL = 30 * 1000;  // 30s
   public static final boolean DEFAULT_LINEAR_RETRY_FOR_CONNECTION_TIMEOUT_ENABLED = true;
+  public static final boolean DEFAULT_STATIC_RETRY_FOR_CONNECTION_TIMEOUT_ENABLED = true;
```

Comment:
Should we make the default value of any one of linear or static as true ?

Reply:
Both the configs need to exist together. I will be using the if else ladder to check these configs..
If config for static is set to true, it does not matter what we have set for linear. It won't be checked.

If config for static is set to false, then only it will check config for linear.
There could be in total 4 combinations:

static   linear    result
true     true      StaticRetryPolicy
true     false     StaticRetryPolicy
false    true      LinearRetryPolicy
false    false     ExponentialRetryPolicy

Reply:
Also, we have decided earlier to keep these configs true by default as long as we are testing out things and doing performance comparisons. 

We can revisit these default values before merging into trunk based on our analysis

Reply:
> 

Lets add a test where in if-else if tested and if someone disturbes it as code-change, the test shall fail. Thanks.

Reply:
Checked again, you already have test. Great work!
Have some comments there.
Thanks.